### ISSUE TITLE
Downtown cab co.

### ISSUE THREAD

**Issue by Zoraketh** [fkk-cz/noire-public#42]
Have had issues with not being able to interact with taxi menu and at the place you get taxi cabs.  It has happened twice, once randomly and I do not know what triggered it, another time was after I died. I can no longer spawn a taxi or try to receive a job through the f6 menu.

**Comment by fkk-cz** [comment]
You're hitting E and the taxi is not despawning?

**Comment by Zoraketh** [comment]
Correct, I could not use any function for that job. I can't remember if I need to fly out or go back to the job center and swap jobs to fix it.
